frontend/model/contracts/chatroom.js

```diff
@@ -114,6 +114,15 @@ function messageReceivePostEffect ({
   shouldSoundMessage && sbp('okTurtles.events/emit', MESSAGE_RECEIVE)
 }
 
+async function deleteEncryptedFiles (manifestCids: string | string[], option: Object) {
+  if (Object.values(option).reduce((a, c) => a || c, false)) {
+    if (typeof manifestCids === 'string') {
```

Comment:
Perhaps not critical, but I'd use `!Array.isArray(manifestCids)` in this case because it'll break more consistently if manifestCids is not a string.

Also, I've pretty much given up at this point on this, but, in JS, `typeof x === 'string'` doesn't work for Object-wrapped strings (e.g., strings from `new String('foo')` or `Object('foo')`). This is mostly academic as such strings are rare and the 'proper' check for this is ugly and doesn't play nice with TypeScript. However, this is in part why if I can avoid `typeof`, as in this case, I do.

Reply:
Reasonable. I will update it now.